pkg/worker/handler/operator/cloudformation/ensure.go

```diff
@@ -0,0 +1,13 @@
+package cloudformation
+
+func (h *CloudFormation) Ensure() error {
+
+	// TODO
+	//
+	//     check for drift
+	//     apply update, if any
+	//     emit deployment event, if updated
+	//
+
+	return nil
+}
```

Comment:
I added this dummy to work around a naming conflict with the internal dependencies. So now we already have this structure in place. Will address those TODOs tomorrow.